Data-Structures/HashTable/HashTable.js

```diff
@@ -0,0 +1,105 @@
+export default class HashTable {
+  constructor(limit = 100) {
+    // Initialize the storage and limit variables
+    this.storage = new Array(limit)
+    this.limit = limit
+  }
+
+  // Hash function:
+  _hash(key, max) {
+    // Initialize the hash variable to 0
+    let hash = 0
+    // Iterate through the key and add the character code at each iteration to the hash variable
+    for (let i = 0; i < key.length; i++) hash += key.charCodeAt(i)
+    // Return the hash modulo the max
+    return hash % max
+  }
+
+  // Insert a key-value pair into the hash table
+  set(key, value) {
+    // Hash the key
+    const index = this._hash(key, this.limit)
+    // If the index is empty, insert the key-value pair and create a bucket
+    if (this.storage[index] === undefined) this.storage[index] = [[key, value]]
```

Comment:
Use an object `{key, value}` here for better readability (s.t. instead of `[0]` and `[1]`, you can do `.key` and `.value` respectively). You could consider introducing a class `Entry` or `Pair` that stores a key-value pair for this.